release_notes/ocp-4-2-release-notes.adoc

```diff
@@ -1563,3 +1563,79 @@ the upgrade is completed.
 * Certain features like the name of the application and the build status are not rendered in the *Topology* view on the Edge browser. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1760858[*BZ#1760858*])
 
 * Determination of active Pods when a rollout fails can be incorrect in the *Topology* view. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1760828[*BZ#1760828*])
+
+[id="ocp-4-2-asynchronous-errata-updates"]
+== Asynchronous errata updates
+
+Security, bug fix, and enhancement updates for {product-title} 4.2 are released
+as asynchronous errata through the Red Hat Network. All {product-title} 4.2
+errata is https://access.redhat.com/downloads/content/290/[available on the Red
+Hat Customer Portal]. See the
+https://access.redhat.com/support/policy/updates/openshift[{product-title}
+Life Cycle] for more information about asynchronous errata.
+
+Red Hat Customer Portal users can enable errata notifications in the account
+settings for Red Hat Subscription Management (RHSM). When errata notifications
+are enabled, users are notified via email whenever new errata relevant to their
+registered systems are released.
+
+[NOTE]
+====
+Red Hat Customer Portal user accounts must have systems registered and consuming
+{product-title} entitlements for {product-title} errata notification
+emails to generate.
+====
+
+This section will continue to be updated over time to provide notes on
+enhancements and bug fixes for future asynchronous errata releases of
+{product-title} 4.2. Versioned asynchronous releases, for example with the form
+{product-title} 4.2.z, will be detailed in subsections. In addition, releases
+in which the errata text cannot fit in the space provided by the advisory will
+be detailed in subsections that follow.
+
+[IMPORTANT]
+====
+For any {product-title} release, always review the instructions on
+xref:../updating/updating-cluster.adoc#updating-cluster[updating your cluster]
+properly.
+====
+
+[[ocp-4-2]]
```

Comment:
I would update this target to be more specific; maybe [[rhba-2019-2921]] or something like that

Reply:
Oh, I see now you have a different schema going in the ones that follow, but ocp-4-2 still seems like it could be used anywhere. Maybe ocp-4-2-0?

Reply:
Agreed! Fixed.